sentence_transformers/sparse_encoder/SparseEncoder.py

```diff
@@ -374,7 +526,9 @@ def _load_auto_model(
         Returns:
             List[nn.Module]: A list containing the transformer model and the pooling model.
         """
-        logger.warning(f"No sparse-encoder model found with name {model_name_or_path}. Creating a new one.")
+        logger.warning(
+            f"No sparse-encoder model found with name {model_name_or_path}. Creating a new one with defaults settings compatible to the based model."
```

Comment:
```suggestion
            f"No sparse-encoder model found with name {model_name_or_path}. Creating a new one with defaults settings compatible to the base model."
```
haha, kind of a funny typo